yabt/fs_global_cache.py

```diff
@@ -43,18 +43,28 @@ def __init__(self, directory='/tmp/cache'):
     def has_cache(self, target_hash: str):
         return isdir(join(self.targets_dir, target_hash))
 
-    def download_summary(self, target_hash: str, dst: str):
+    def download_summary(self, target_hash: str, dst: str) -> bool:
```

Comment:
now that you support "random erasure", you can add a test which:
- erases 10% of summaries
- erases 10% of artifacts
and still the cache magically downloads what it's supposed to.